pkg/kubelet/volumes.go

```diff
@@ -106,11 +99,13 @@ func (vh *volumeHost) GetWriter() io.Writer {
 	return vh.kubelet.writer
 }
 
-// Returns the hostname of the host kubelet is running on
```

Comment:
@swagiaal @pmorie  -  gottcha, and that makes sense.  I added back for now, unless Paul has a better reason to remove it.